### PR TITLE
Update Kythera setup guide to use the project centric workflow

### PR DESCRIPTION
**Pull request by fabioanderegg**
We will release an updated version of the Gem shortly that works with an installed/SDK Engine. This new installation process is much easier and faster.

### CODE REVIEW

File: content/docs/user-guide/gems/reference/kythera-ai/kythera-ai-gem-setup.md [hunk 1]
```diff
@@ -8,43 +8,34 @@ toc: true
...
-    cmake -B MyProject/build -S MyProject -DLY_EXTERNAL_SUBDIRS=<path to unpacked zip>\Gems\Kythera`
-    ```
-
-1. Rebuild the project from within Visual Studio.
+Make sure you have a working engine installation from the installer or an SDK installation as documented here: [setup guide](/docs/welcome-guide/setup/).
+
+1. Unpack the zip file to some folder
+1. Open the O3DE Project Manager (`o3de.exe`)
+1. Go to `New Project...` and select `Add existing Project`
+1. Select the KytheraAIDemo folder in the unpacked zip folder
```
Comment (sptramer) [inline content/docs/user-guide/gems/reference/kythera-ai/kythera-ai-gem-setup.md:20]:
```suggestion
1. Select the `KytheraAIDemo` folder in the unpacked zip folder.
```

File: content/docs/user-guide/gems/reference/kythera-ai/kythera-ai-gem-setup.md [hunk 2]
```diff
@@ -8,43 +8,34 @@ toc: true
...
+1. Unpack the zip file to a folder
+1. Open a command prompt/PowerShell window inside your engine installation
+1. Run the following command: `.\scripts\o3de.bat register -gp <path to the Kythera Gem> --external-subdirectory-project-path <path to your game directory>`. This will add the `external_subdirectories` key to the `project.json` file and make the Gem available to your project.
+1. Open the O3DE Project Manager (`o3de.exe`)
+1. In the project menu, click `Edit Project Settings` and then the `Configure Gems` button on the top right corner
+1. Kythera AI should now be selectable as a Gem. Select it and save the project settings.
+1. There is an outstanding issue in the Engine that the Editor component of Kythera is not built automatically when using and SDK build, so it has to be done manually for now:
+    1. Open the project's Visual Studio solution (`<project directory>/build/windows_vs2019/<project name>.sln`)
+    1. Switch build type from `debug` to `profile`
+    1. Build the target `Kythera.Editor`, which is in one of the subdirectories matching the full filesystem path to the Gem
```
Comment (sptramer) [inline content/docs/user-guide/gems/reference/kythera-ai/kythera-ai-gem-setup.md:41]: See above.

```suggestion
1. Open the project's Visual Studio solution (`<project directory>/build/windows_vs2019/<project name>.sln`).
1. Switch the build type from `debug` to `profile`.
1. Build the target `Kythera.Editor`, which is in one of the subdirectories matching the full filesystem path to the Gem.
```

File: content/docs/user-guide/gems/reference/kythera-ai/kythera-ai-gem-setup.md [hunk 3]
```diff
@@ -8,43 +8,34 @@ toc: true
...
+1. Back in the O3DE launcher, click `Open Editor`
+1. After the Editor has launched, make sure that all assets have been processed in the Asset Processor (Status is Idle), otherwise opening our demo levels will throw errors
+
+# Add the Kythera AI Gem to a project
+The following steps work for both installed/SDK builds and engine centric builds:
+1. Unpack the zip file to a folder
+1. Open a command prompt/PowerShell window inside your engine installation
+1. Run the following command: `.\scripts\o3de.bat register -gp <path to the Kythera Gem> --external-subdirectory-project-path <path to your game directory>`. This will add the `external_subdirectories` key to the `project.json` file and make the Gem available to your project.
+1. Open the O3DE Project Manager (`o3de.exe`)
+1. In the project menu, click `Edit Project Settings` and then the `Configure Gems` button on the top right corner
```
Comment (sptramer) [inline content/docs/user-guide/gems/reference/kythera-ai/kythera-ai-gem-setup.md:36]:
```suggestion
1. Open the O3DE Project Manager (`o3de.exe`).
1. In the project menu, click `Edit Project Settings` and then the `Configure Gems` button on the top right corner.
```

**Review by sptramer — CHANGES_REQUESTED** [review]
Changes requested due to style issues throughout.

Please review the [O3DE Style Guide](https://o3de.org/docs/contributing/to-docs/style-guide/) before making future pull requests or changes.

File: content/docs/user-guide/gems/reference/kythera-ai/kythera-ai-gem-setup.md [hunk 4]
```diff
@@ -8,43 +8,34 @@ toc: true
...
+1. Unpack the zip file to some folder
+1. Open the O3DE Project Manager (`o3de.exe`)
+1. Go to `New Project...` and select `Add existing Project`
+1. Select the KytheraAIDemo folder in the unpacked zip folder
+1. In the project menu, click `Build`<br>
+![Project Menu Build](/images/user-guide/gems/kythera-ai/project-manager-project-menu-build.png)
+1. There is an outstanding issue in the Engine that the Editor component of Kythera is not built automatically, so it has to be done manually for now:
+    1. Open the generated Visual Studio solution `<unpacked zip>/KytheraAIDemo/build/windows_vs2019/KytheraAIDemo.sln`
+    1. Switch build type from `debug` to `profile`
+    1. Build the target `Kythera.Editor`, which is in one of the subdirectories matching the full filesystem path to the Gem
```
[the remark this answers is not in the document]
Comment (fabioanderegg) [reply content/docs/user-guide/gems/reference/kythera-ai/kythera-ai-gem-setup.md:26]: The issue has been fixed yesterday in the development branch. I'll check with the team if we can get the fix into the stabilization branch and completely remove the section (maybe with a note that users need a recent version of the engine).

File: content/docs/user-guide/gems/reference/kythera-ai/kythera-ai-gem-setup.md [hunk 5]
```diff
@@ -8,43 +8,34 @@ toc: true
...
-1. Run the cmake configuration command and add the Kythera Gem directory as an external subdirectory:
-
-    ```cmd
-    cmake -B MyProject/build -S MyProject -DLY_EXTERNAL_SUBDIRS=<path to unpacked zip>\Gems\Kythera`
-    ```
-
-1. Rebuild the project from within Visual Studio.
+Make sure you have a working engine installation from the installer or an SDK installation as documented here: [setup guide](/docs/welcome-guide/setup/).
+
+1. Unpack the zip file to some folder
```
[the remark this answers is not in the document]
Comment (fabioanderegg) [reply content/docs/user-guide/gems/reference/kythera-ai/kythera-ai-gem-setup.md:17]: The name of the zip file changes with every release, but I think I can fix the sentence in another way.

File: content/docs/user-guide/gems/reference/kythera-ai/kythera-ai-gem-setup.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (fabioanderegg) [reply content/docs/user-guide/gems/reference/kythera-ai/kythera-ai-gem-setup.md:27]: Issue has been merged in both development and stabilization, so I will remove this section.

**Comment by fabioanderegg** [comment]
Still struggling with the inline image. @sptramer your suggested change makes the list count from 1 again, and the image is not indented correctly.

I tried all kinds of stuff:
```
1. In the project menu, select **Build**.

    ![Project Menu Build](/images/user-guide/gems/kythera-ai/project-manager-project-menu-build.png)

1. Back in the O3DE launcher, click **Open Editor**.
```
4 spaces + 2 newlines: This pulls the list apart (just as it does here: http://localhost:1313/docs/user-guide/project-config/add-remove-gems/)

Other stuff like adding a newline after the image starts the list again.

Any ideas?

**Comment by sptramer** [comment]
LGTM, and the image does render correctly in the staging site as indented/inline with the list.

**Pull request merged**